src/components/Integrations/Form/IntegrationTypeForm.tsx

```diff
@@ -17,7 +17,8 @@ export const IntegrationTypeForm: React.FunctionComponent<IntegrationTypeForm> =
     if (isCamelType(props.type)) {
         switch (props.type) {
             case UserIntegrationType.SPLUNK:
-                return <IntegrationTypeSplunkForm { ...props } />;
+            case UserIntegrationType.SERVICE_NOW:
+                return <IntegrationTypeCamelExtrasForm { ...props } />;
```

Comment:
Actually the form mentions Splunk:
![183751060-6ece0ba6-b593-4eed-8b8e-63eb89bc7ff4](https://user-images.githubusercontent.com/7695766/183863430-97b2f503-8648-4cf8-9a64-9e01e7d721b0.png)

The secret token would for ServiceNow be a user password. Maybe we can generalize the wording for ServiceNow.

Reply:
No specifics at the moment for SNow. It might be a "Password of a ServiceNow integration user.".

Can we differentiate the text based on the type?

Reply:
yes, we can do that.